.circleci/config.yml

```diff
@@ -1,13 +1,28 @@
-# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
-version: 2.1
-# Use a package of configuration called an orb.
-orbs:
-  # Declare a dependency on the welcome-orb
-  welcome: circleci/welcome-orb@0.4.1
-# Orchestrate or schedule a set of jobs
-workflows:
-  # Name the workflow "welcome"
-  welcome:
-    # Run the welcome/run job in its own container
-    jobs:
-      - welcome/run
\ No newline at end of file
+version: 2.0
+
+jobs:
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
```

Comment:
You have mentioned that this is not needed probably right?

Reply:
I totally forgot about it. Let me fix

Reply:
I'm using this text as guide, I hope the version matches 
https://circleci.com/blog/build-cicd-piplines-using-docker/